sr/comp/mixtape/mixtape.py

```diff
@@ -23,12 +24,33 @@ def __init__(
         playlist: Any,
         audio_controller: AudioController,
         magicq_controller: Optional[MagicqController],
+        obs_studio_controller: Optional[OBSStudioController],
     ) -> None:
         self.root = root
         self.playlist = playlist
         self.audio_controller = audio_controller
         self.exclusivity_groups: Dict[object, subprocess.Popen[bytes]] = {}
         self.magicq_controller = magicq_controller
+        self.obs_studio_controller = obs_studio_controller
+
+    def get_play_video_action(
+        self,
+        track: Any,
+        current_offset: Callable[[], float],
+    ) -> Tuple[Action, str]:
+        path = os.path.join(self.root, track['obs_video'])
```

Comment:
By converting this into an absolute path we are forcing mixtape to run on the same computer as OBS. Ultimately leaving this as an OBS-relative path gives us the option to separate the services.

Reply:
Have clarified this by removing the host option.